src/app.tsx

```diff
@@ -7,33 +7,36 @@ import {
   SafeAreaProvider,
   initialWindowMetrics,
 } from 'react-native-safe-area-context';
-import { CatContextProvider } from './contexts';
+import { CatContextProvider, store } from './contexts';
 import { ErrorFallback } from './components';
-import { ThemeProvider } from '@rneui/themed';
 import appTheme from './app-theme';
 import { DatadogProvider } from '@datadog/mobile-react-native';
 import DatadogConfig from './services/datadog';
 import Logger from './logger/logger';
+import { NativeBaseProvider } from 'native-base';
```

Comment:
If possible - We should look around for a eslint rule to put restriction on ordering as well as please. Module imports (native-base, react-redux) should come before files. We can also create an issue and address it later.

Reply:
Sure, will create an issue regarding this and address it.